config/views.yaml

```diff
@@ -1476,3 +1476,47 @@ component_readiness:
       enabled: true
     regression_tracking:
       enabled: true
+  - name: coo-component-readiness
```

Comment:
This should likely have a release specified in it.  Also consider the existing [4.21-LP-Interop](https://github.com/IshwarKanse/sippy/blob/ac9b02316ea0dedbbbda00e1644ee275ed6f6cfa/config/views.yaml#L293), would it make sense to be `4.19-LP-Interop` or do you need it to be specific to `4.19-lp-Interop-coo`

Reply:
We need 4.19-lp-Interop-coo as this will be specifically used by the COO only.